docs/learn/learn-staking-faq.md

```diff
@@ -0,0 +1,68 @@
+---
+id: learn-staking-faq
+title: Staking FAQ
+sidebar_label: Staking FAQ
+description: Addresses FAQ on Staking
+slug: ../learn-staking-faq
+---
+
+
+# Staking FAQ
+
+## 1. How do I stake my DOT?
+
+Thanks for considering to nominate and stake your DOT to secure the Polkadot Network and its parachains. The success of Nominated Proof of Staking (NPoS) relies on well informed nominators. To directly submit your intent to stake on Polkadot, you should
+
+-  have an account on a [non-custodial wallet](../build/build-wallets.md) that supports staking. If you are a beginner, watch this tutorial on [creating an account on Polkadot-JS Extension](https://youtu.be/sy7lvAqyzkY).
```

Comment:
Section 3 sufficiently covers why someone might get excluded, but in the future I might consider adding another bullet point here along the lines of "make sure you have at least the min-active stake bonded". Although in theory, if there is a huge slashing event this could drop significantly and waiting nominators could suddenly make it in

Reply:
I agree. I will add that info when we create a tutorial showing how to determine that value using the UI